LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) HTML5 Boilerplate
```

Comment:
I highly doubt they have the © for this website. Maybe for the boilerplate, but not more.